melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/helpers/SedMapperFactory.java

```diff
@@ -1,39 +1,53 @@
 package no.nav.melosys.eessi.service.sed.helpers;
 
+import java.lang.reflect.InvocationTargetException;
+import java.util.Map;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
 import lombok.experimental.UtilityClass;
 import no.nav.melosys.eessi.models.SedType;
 import no.nav.melosys.eessi.models.exception.MappingException;
 import no.nav.melosys.eessi.service.sed.mapper.SedMapper;
-import no.nav.melosys.eessi.service.sed.mapper.horisontal.H005Mapper;
+import no.nav.melosys.eessi.service.sed.mapper.horisontal.*;
 import no.nav.melosys.eessi.service.sed.mapper.lovvalg.*;
 
-import java.lang.reflect.InvocationTargetException;
-import java.util.EnumMap;
-import java.util.Map;
-
 @UtilityClass
 public class SedMapperFactory {
 
-    static Map<SedType, Class<? extends SedMapper>> sedMappers = new EnumMap<>(SedType.class);
+    static final Map<SedType, Class<? extends SedMapper>> SED_MAPPERS =
+            Maps.immutableEnumMap(ImmutableMap.<SedType, Class<? extends SedMapper>>builder()
+                    .put(SedType.A001, A001Mapper.class)
+                    .put(SedType.A002, A002Mapper.class)
+                    .put(SedType.A003, A003Mapper.class)
+                    .put(SedType.A005, A005Mapper.class)
+                    .put(SedType.A008, A008Mapper.class)
+                    .put(SedType.A009, A009Mapper.class)
+                    .put(SedType.A010, A010Mapper.class)
+                    .put(SedType.A011, A011Mapper.class)
 
-    static {
-        sedMappers.put(SedType.A001, A001Mapper.class);
-        sedMappers.put(SedType.A002, A002Mapper.class);
-        sedMappers.put(SedType.A003, A003Mapper.class);
-        sedMappers.put(SedType.A005, A005Mapper.class);
-        sedMappers.put(SedType.A008, A008Mapper.class);
-        sedMappers.put(SedType.A009, A009Mapper.class);
-        sedMappers.put(SedType.A010, A010Mapper.class);
-        sedMappers.put(SedType.A011, A011Mapper.class);
-        sedMappers.put(SedType.H005, H005Mapper.class);
-    }
+                    .put(SedType.H001, H001Mapper.class)
```

Comment:
Vi kunne jo egentlig bare instansiert klassene her, da hadde vi også sluppet å ha en klasse per sed-type for SED-er som vi kun støtter preutfylling for. Eks `.put(SedType.H001, new HSedMapper(SedType.H001))`

Da hadde vi og blitt kvitt refleksjon ved instansiering av mapperen